FirebaseAuth/Tests/SampleSwift/SwiftApiTests/EmailPasswordTests.swift

```diff
@@ -69,12 +69,15 @@ class EmailPasswordTests: TestsBase {
     waitForExpectations(timeout: TestsBase.kExpectationsTimeout)
   }
 
-  @available(iOS 13, tvOS 13, macOS 10.15, macCatalyst 13, watchOS 7, *)
   func testSignInExistingUserWithEmailAndPasswordAsync() async throws {
     let auth = Auth.auth()
     try await auth.signIn(withEmail: kExistingEmailToSignIn, password: "password")
     XCTAssertEqual(auth.currentUser?.email,
                    kExistingEmailToSignIn,
                    "Signed user does not match request.")
+    // Regression test for #13550. Auth enumeration protection is enabled for
```

Comment:
Should it be "Auth enumeration protection is **not** enabled for the test project **so no** sign in methods should be returned."

Reply:
Not quite, but: Auth enumeration protection is ~not~ enabled for the test project so no sign in methods should be returned.

https://github.com/firebase/firebase-ios-sdk/blob/486a0a03c197a267a4a54d5f0077e1703b197588/FirebaseAuth/Sources/Swift/Auth/Auth.swift#L286